src/wasm.h

```diff
@@ -71,6 +71,7 @@ struct Name : public cashew::IString {
   Name() : cashew::IString() {}
   Name(const char *str) : cashew::IString(str) {}
   Name(cashew::IString str) : cashew::IString(str) {}
+  Name(const std::string &str) : cashew::IString(str.c_str()) {}
```

Comment:
Ah OK, didn't know that. Sounds a bit dangeous to have `const char *` not have the same thing?